public/index.html

```diff
@@ -316,11 +316,17 @@
     <div class="section">
       <h2>Events</h2>
 
-      <p>All events are called on original element, first argument is the original element too. And can be bound like this:</p>
-
-      <pre><code class="language-javascript">$('select').on('eventname', function(element) {
-&nbsp;&nbsp;// your code
-});</code></pre>
+      <p>All events are called on original element. Within the callback you have access to the <code>event object</code>, the original
+        <code>select element</code> and to the current <code>selectric instance</code>. It can be bound like this:</p>
+
+      <pre><code class="language-javascript">/**
+ * @param {Object}          event - jquery event object
+ * @param {HTMLElement}   element - original dom element
+ * @param {Object}      selectric - current selectric instance
+ */
+ $('select').on('eventname', function(event, element, selectric) {
```

Comment:
@lcdsantos We should consider to reduce the complexity a bit here and remove the parameter `element`. There is already an reference inside the selectric object `selectric.$element`. This should be enough, don't you think?